packages/devtools_app/lib/src/screens/inspector/inspector_service.dart

```diff
@@ -433,6 +433,16 @@ class InspectorService extends InspectorServiceBase {
     await _onRootDirectoriesChanged(rootDirectories);
```

Comment:
remove setPubRootDirectories and _setPubRootDirectories methods since they will no longer be in use